src/styles/index.scss

```diff
@@ -1,3 +1,219 @@
 body {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+$primary-color: #00acdc;
+$secondary-color: #616070;
+$text-color: #060b35;
+
+.header {
+  display: flex;
+  height: 60px;
+  border-bottom: 1px solid #ddd;
+
+  &__nav {
+    padding: 22px 50px 23px;
+    display: flex;
+    align-items: center;
+  }
+
+  &__logo {
+    width: 40px;
+    height: 40px;
+    top: 10px;
+    left: 50px;
+  }
+
+  &__nav-list {
+    padding-left: auto;
+    list-style: none;
+    display: flex;
+    margin-left: 333px;
+    gap: 20px;
+  }
+
+  &__nav-link {
+    text-transform: uppercase;
+    color: $text-color;
+    text-decoration: none;
+    line-height: 14.06px;
+    font-size: 12px;
+    font-weight: 500;
+
+    &:first-child {
+      margin-left: px;
```

Comment:
The 'margin-left' property is missing a value. It currently has 'px' without a numerical value, which is incorrect. Please provide a valid numerical value for this property.